pkg/liqoctl/generate/consts.go

```diff
@@ -18,7 +18,10 @@ package generate
 const LiqoctlGenerateShortHelp = "Generate the command to execute on another cluster to peer with the selected cluster."
 
 // LiqoctlGenerateLongHelp contains the short help message for the generate-add-cluster Liqoctl command.
-const LiqoctlGenerateLongHelp = "Generate the command to execute on another cluster to peer with the selected cluster."
+const LiqoctlGenerateLongHelp = `Generate the command to execute on another cluster to peer with the selected cluster.
+
+liqoctl generate-add-command command requires Liqo to be installed on the target cluster.
```

Comment:
To me, this sentence looks misleading, as I feel you are referring to the cluster you want to peer with (target), and not the one you are executing this command on.